src/coordination/azure/Akka.Coordination.Azure/AzureLease.cs

```diff
@@ -127,53 +127,57 @@ public override Task<bool> Acquire(Action<Exception?>? leaseLostCallback)
                 if (_acquireTask is not null)
                 {
                     if(_log.IsDebugEnabled)
-                        _log.Debug("Lease is already being acquired");
+                        _log.Debug("Lease [{0}] for [{1}] is already being acquired", _leaseName, _settings.OwnerName);
                     return _acquireTask;
                 }
 
                 if(_log.IsDebugEnabled)
-                    _log.Debug("Acquiring lease");
-                _acquireTask = _leaseActor.Ask(new LeaseActor.Acquire(leaseLostCallback), _timeout)
-                    .ContinueWith(t =>
```

Comment:
Refactor `.ContinueWith()` to internal method, cleaner code makes it easier to debug.

Reply:
LGTM